website/content/en/docs/best-practices/common-recommendation.md

```diff
@@ -76,7 +76,7 @@ spec:
 - Use [finalizers][finalizers] when/if required 
 - Cover the project with tests/CI to ensure its quality:
     - For any language-based operator, you can use [Scorecard][scorecard] to implement functional tests
-    - For Go-based operators, you can also use [envtest][envtest] to cover the controllers. For further information see [Testing with EnvTest][testing-with-envtest]
+    - For Go-based operators, you can also use [envtest][envtest] to cover the controllers. For further information see [Testing with EnvTest][testing-with-envtest]. Also, see the `test` directory for the Memcached sample under the [testdata/go/v3/memcached-operator][sample] to know how can you built e2e tests.
```

Comment:
```suggestion
    - For Go-based operators, you can also use [envtest][envtest] to cover the controllers. For further information see [Testing with EnvTest][testing-with-envtest]. Also, see the `test` directory for the Memcached sample under the [testdata/go/v3/memcached-operator][sample] to know how can you build e2e tests.
```

Reply:
@camilamacedo86 Here is where I meant we could explain how we are setting up envtest. Similar to https://book.kubebuilder.io/reference/envtest.html#environment-variables. The KB doc does not mention about installing cert-manager and prometheus. We could add that here as an extension telling why we do so, and how we test webhooks and a running operator. This could be done in a follow up too.

Reply:
@varshaprasad96 that is agreat point. I think it needs to be in the kb doc right since are options provided by default. I tracked the need in:  https://github.com/kubernetes-sigs/kubebuilder/issues/2777.

But sure we can improve in follow-ups these points here as well.